### ISSUE TITLE
System Down View web component: development

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/va.gov-team#37733]
## Description
Build a web-component version of our [SystemDownView React component](https://github.com/department-of-veterans-affairs/component-library/tree/master/packages/react-components/src/components/SystemDownView).

## Acceptance Criteria
- [ ] `va-system-down-view` is written
- [ ] E2E tests are written
- [ ] Add Chromatic link to #36837, assign ticket to Courtney, and move ticket to Current Sprint column
- [ ] Once accessibility review is complete, alert Carol that component is ready for staging review
- [ ] Address any staging review comments
- [ ] Merge component

**Comment by caw310** [comment]
Platform teams have been asked to swarm on initiative work. During refinement today, we discussed which web components to build in Q2. Decision is to focus on new components that would be useful to the forms library so removing from sprint 75 to be worked on in the future. Possibly as a stretch goal in Q2 2022.

**Comment by caw310** [comment]
Closing as this is old and the VADS has been updated.
